### ISSUE TITLE
Why repo not in ublue-os organization?

### ISSUE THREAD

**Issue by plata** [NiHaiden/aurora-web#17]
The project code itself is part of the ublue-os organization (https://github.com/ublue-os/bluefin) so why not the website as well? It would simplify continuity in case e.g. you cannot maintain the website anymore for whatever reason.

**Comment by nicknamenamenick** [comment]
Yeah the websites for the images (like Bazzite) are still under the lead maintainer's repo as opposed to ublue-os right now.  Eventually, these should be moved but it's more of a later thing.  Sorry if this was difficult to find.
